### ISSUE TITLE
Site does not use Bootstrap CSS

### ISSUE THREAD

**Issue by A1Liu** [BUGS-NYU/bugs-nyu.github.io-old#86]
The site has bootstrap javascript, but does not include the CSS from the website. This means it's not taking advantage of all the powerful stuff that bootstrap offers.

**Comment by A1Liu** [comment]
Suggested fix:
paste
```html
<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
```
into the file `header.html` located in the `_includes` folder, in the line above the other stylesheet (right underneath `<title> BUGS - NYU </title>`)

**NOTE:** This may cause some issues. For example, the about page uses the `col-lg-4` class to format the users, which is a namespace included in bootstrap. Thus, when converting to bootstrap, the about page may look different.

**Comment by A1Liu** [comment]
Fixed by #124 .
